util.go

```diff
@@ -108,6 +108,11 @@ func (c *Config) updateSnapshot(snapshotName string, snapshot string) error {
 		return fmt.Errorf("snapshot created for test %s", snapshotName)
 	}
 
+	if !c.failOnUpdate {
```

Comment:
placement of this `if` block here means an error will still be returned when `isNewSnapshot`? Could this be placed before the `if` block above?

Reply:
Ack, you're right, a new snapshot should be classed the same as an update. I'll move this above the isNewSnapshot check